src/cloudant/design_document.py

```diff
@@ -498,7 +498,12 @@ def fetch(self):
                         view_def.pop('reduce', None),
                         **view_def
                     )
-
+        # dbcopy is transformed to a new storage location, we need to de-transform
+        if 'options' in self:
+            if 'epi' in self['options']:
+                if 'dbcopy' in self['options']['epi']:
+                    for view in self['options']['epi']['dbcopy'].keys():
+                        self['views'][view]['dbcopy'] = self['options']['epi']['dbcopy'][view]
```

Comment:
Yes, that is what I'm suggesting, since someone who is new to the client and using DBNext, (which is most on MT now IIRC) will expect there to only be in the new location when creating a new dbcopy view.  So I would expect it to be copied if a config option was set to make it appear as if dpcopy was in the old location.